src/app/frontend/replicasetlist/replicasetlist_controller.js

```diff
@@ -19,11 +19,21 @@
  */
 export default class ReplicaSetListController {
   /**
+   * @param {!ui.router.$state} $state
    * @param {!backendApi.ReplicaSetList} replicaSets
    * @ngInject
    */
-  constructor(replicaSets) {
+  constructor($state, replicaSets) {
     /** @export {!Array<!backendApi.ReplicaSet>} */
     this.replicaSets = replicaSets.replicaSets;
+
+    /** @private {!ui.router.$state} */
+    this.state_ = $state;
   }
+
+  /**
+   * TODO(floreks): Should be changed to state variable. Related issue #153.
+   * @export
+   */
+  redirectToDeployPage() { this.state_.go('deploy'); }
```

Comment:
Import state name from the `foo_state.js` file. This technique has a nice effect that there are no implicit dependencies between code files - if there is a dependency on something, it is imported and used. Here, the dependency is 'deploy' string, which may be changed, deleted or moved.

Of course, in HTML files we still have implicit deps to directive names, but this is likely to be fixed with Angular 2.0 :)